lib/internal/fs/streams.js

```diff
@@ -183,10 +187,16 @@ ReadStream.prototype._read = function(n) {
       // Now that we know how much data we have actually read, re-wind the
       // 'used' field if we can, and otherwise allow the remainder of our
       // reservation to be used as a new pool later.
-      if (start + toRead === thisPool.used && thisPool === pool)
-        thisPool.used += bytesRead - toRead;
-      else if (toRead - bytesRead > kMinPoolSpace)
-        poolFragments.push(thisPool.slice(start + bytesRead, start + toRead));
+      if (start + toRead === thisPool.used && thisPool === pool) {
+        const alignedOffset = thisPool.used + bytesRead - toRead;
```

Comment:
Tiniest of nits: `alignedOffset` is a misnomer since it's not actually aligned (yet.)